docs/using-guidelines.asciidoc

```diff
@@ -36,6 +50,3 @@ ingesting to Elasticsearch using the API, your fields must be nested
 objects, not strings containing dots.
 
 * *Avoid abbreviations when possible*. A few exceptions like `ip` exist.
```

Comment:
This is repetitive and should be removed or made consistent by including `os` and `geo` etc.